deployments/ooi/config/common.yaml

```diff
@@ -37,6 +37,10 @@ pangeo:
         guarantee: 1.25G
     hub:
       #nodeSelector: {agentpool: nodepool1}
+      services:
+        dask-gateway:
+          # This makes the gateway available at ${HUB_URL}/services/dask-gateway
+          url: "http://web-public-ooi-staging-dask-gateway"
```

Comment:
You'll want "http://web-public-ooi-prod-dask-gateway" as well right? So this should go into both prod.yaml and staging.yaml instead of common.yaml?

Reply:
Ah right good catch. This needs to be split. Thanks!